src/pages/ProfilePage.js

```diff
@@ -137,6 +138,11 @@ function ProfilePage(props) {
 
     const chatReportWithCurrentUser = !isCurrentUser && !Session.isAnonymousUser() ? ReportUtils.getChatByParticipants([accountID]) : 0;
 
+    const notificationPreference =
+        chatReportWithCurrentUser.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN
```

Comment:
We can move this to constant and reuse it in line 240.

Reply:
Good point, DRYed this up.